oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+[#]_), micro-front ends (MFEs [#]_) and libraries.  The code for these
+services and libraries is managed in git and hosted on GitHub
+platform.
+
+Each component part of the Open edX platform requires maintenance to
+ensure that it meets, and continues to meet, the project definition of
+high quality.  That definition will include project standards,
+appropriate documentation, and fitness for use, among other things.
+That definition will evolve as project needs do and we learn what
+works and what doesn't.
+
+Component maintenance is work.  To ensure that that work is done, a
+person or persons must be recognized as responsible for doing it,
+and held accountable for its performance.
+
+The Open edX project, like many projects, calls this role
+"maintainer."  The notion of a project maintainer is typical of
+open-source projects and widely considered an best practice [#]_.
+
+The maintainer role is a leadership role that encompasses both
+technical and community elements.  Both dimensions of the role are
+equally important.
+
+All, non-deprecated, components of the Open edX project must have a
+named maintainer.  When no such maintainer exists, this will be
+considered an urgent problem to solve.
+
+This OEP proposes a formal definition of the role of maintainer that
+will open to all qualified Open edX community members.  Maintainers
+will be responsible for the health of platform components, but not for
+their performance in production [#]_.
+
+Motivation
+==========
+
+With the change in stewardship from edX to The Center for Reimagining
+Learner, it is essential to have a well defined plan for maintaining
+the platform.
+
+In order to ensure that the project is sustainable for the long term,
+maintenance must be shared by the entire community.  The project must
+be robust enough that a it can tolerate firms and contributors leaving
+the ecosystem.
+
+As a project we must acknowledge that maintenance is hard work, but
+work that must be done for the platform to be fit for use.  The best
+way of ensuring this is to have a named person or persons who are
+responsible for getting the work done.
+
+Specification
+=============
+
+Definitions
+-----------
+
+The term **repository** will mean a repository on the GitHub social
+coding platform.
+
+The term **component** will typically mean the unit of software housed
+in a single GitHub repository.  Exceptions will be made in the case of
+edx-platform where there may be multiple distinguishable components in
+that single repository.  Components will typically, but not always, be
+hosted at the Open edX GitHub organization.  Components will be "part
+of" or "critical to" the open-source platform.
+
+The term **maintainer** will mean a named person responsible for the
+health of component and the component's community.  The remainder of
+this specification will be focused on the details of the role.
+
+The term **core contributor** will mean a named person participating
```

Comment:
I feel it's worth specifying that this is a _coding core contributor_ or _core contributor with commit rights_ or something similar.

Maybe that's a nit, but I want to make sure across documentation we're keeping in mind that CC is a global program spanning many roles. The default is not any one particular role, and I'm continuously keeping in mind of not holding up the committer role as the "best" or "most important" or "default".

Reply:
@sarina I've elaborated, but the change is just below so GH doesn't detect this comment as outdated.  Does that new version work for you?

Reply:
Yes, thanks!